sdk/storage/azure-storage-file-share/tests/test_directory.py

```diff
@@ -1396,6 +1399,53 @@ def test_bad_audience_directory_client(self, **kwargs):
         # Assert
         directory_client.exists()
 
+    @FileSharePreparer()
+    @recorded_by_proxy
+    def test_file_permission_format_for_directory_operations(self, **kwargs):
+        storage_account_name = kwargs.pop("storage_account_name")
+        storage_account_key = kwargs.pop("storage_account_key")
+
+        self._setup(storage_account_name, storage_account_key)
+        share_client = self.fsc.get_share_client(self.share_name)
+        directory_client = share_client.create_directory('dir1')
+
+        directory_properties_on_creation = directory_client.get_directory_properties()
+        last_write_time = directory_properties_on_creation.last_write_time
+        creation_time = directory_properties_on_creation.creation_time
+        change_time = directory_properties_on_creation.change_time
+
+        new_last_write_time = last_write_time + timedelta(hours=1)
+        new_creation_time = creation_time + timedelta(hours=1)
+        new_change_time = change_time + timedelta(hours=1)
```

Comment:
No need to do this extra work with the properties we don't care about for this test. I would remove all this and focus on permissions.